pandas/tests/indexes/multi/test_drop.py

```diff
@@ -147,3 +147,22 @@ def test_drop_with_nan_in_index(nulls_fixture):
     msg = r"labels \[Timestamp\('2001-01-01 00:00:00'\)\] not found in level"
     with pytest.raises(KeyError, match=msg):
         mi.drop(pd.Timestamp("2001"), level="date")
+
+
+def test_single_level_drop():
```

Comment:
Could you rename to something like test_single_level_drop_partially_missing_elements?

Reply:
@GYHHAHA can you do this one, ping on green.